src/software/ai/hl/stp/play/ball_placement_play_test.py

```diff
@@ -12,92 +12,97 @@
 from proto.ssl_gc_geometry_pb2 import Vector2
 
 # TODO (#2599): Remove Duration parameter from test
-@pytest.mark.parametrize(
-    "run_enemy_ai,test_duration", [(False, 20)]
-)  # , (True, 20)]) # TODO (#2690): Robot gets stuck in corner of defense area
-def test_two_ai_ball_placement(simulated_test_runner, run_enemy_ai, test_duration):
+# TODO (#2690): Robot gets stuck in corner of defense area
 
-    # starting point must be Point
-    ball_initial_pos = tbots.Point(2, 2)
-    # placement point must be Vector2 to work with game controller
-    ball_final_pos = tbots.Point(-3, -2)
-
-    # Setup Bots
-    blue_bots = [
-        tbots.Point(-2.75, 2.5),
-        tbots.Point(-2.75, 1.5),
-        tbots.Point(-2.75, 0.5),
-        tbots.Point(-2.75, -0.5),
-        tbots.Point(-2.75, -1.5),
-        tbots.Point(4.6, -3.1),
-    ]
 
-    yellow_bots = [
-        tbots.Point(1, 0),
-        tbots.Point(1, 2.5),
-        tbots.Point(1, -2.5),
-        tbots.Field.createSSLDivisionBField().enemyGoalCenter(),
-        tbots.Field.createSSLDivisionBField().enemyDefenseArea().negXNegYCorner(),
-        tbots.Field.createSSLDivisionBField().enemyDefenseArea().negXPosYCorner(),
-    ]
-
-    # Game Controller Setup
-    simulated_test_runner.gamecontroller.send_ci_input(
-        gc_command=Command.Type.STOP, team=Team.UNKNOWN
-    )
-    simulated_test_runner.gamecontroller.send_ci_input(
-        gc_command=Command.Type.FORCE_START, team=Team.BLUE
-    )
-    # Pass in placement point here - not required for all play tests
-    simulated_test_runner.gamecontroller.send_ci_input(
-        gc_command=Command.Type.BALL_PLACEMENT,
-        team=Team.BLUE,
-        final_ball_placement_point=ball_final_pos,
-    )
+def test_two_ai_ball_placement(simulated_test_runner):
 
-    # Force play override here
-    blue_play = Play()
-    blue_play.name = PlayName.BallPlacementPlay
+    # placement point must be Vector2 to work with game controller
+    ball_final_pos = tbots.Point(-3, -2)
 
-    simulated_test_runner.blue_full_system_proto_unix_io.send_proto(Play, blue_play)
+    def test_setup(run_enemy_ai):
```

Comment:
let's rename this to `setup` because rn test_setup suggests that we're testing the setup

Reply:
done